modules/performance/extension/siddhi-execution-performance/component/src/main/java/org/wso2/extension/siddhi/execution/throughput/CalculatePerformanceStreamProcessorExtension.java

```diff
@@ -68,6 +68,15 @@
                 @Parameter(name = "iijtimestamp",
                            description = "This value used to find the sending timestamp from client",
                            type = {DataType.LONG}),
+
+                           name = "Count",
+                           description = "This value used to track the count of throughput extension calling",
```

Comment:
This sentence "This value used to track the count of throughput extension calling" does not have meaning. What do you mean by count of throughput extension calling?